File: oddeven.c
#include<stdio.h>
#include<Conio.h>

int main(){
    int number;
    printf("enter your number");
    scanf("%d",&number);
    if(number%2==0){
        printf("its an even number");
    }
    else{
        printf("its is an odd number");
    }
    return 0;
}

File: oddeven.java

import java.util.Scanner;

public class oddeven {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        System.out.print("Enter the size of array:");
        int size =sc.nextInt();
        int[]arr=new int[size];
        for (int i = 0; i < arr.length; i++) {
            System.out.print("enter the elements of the array:");
            arr[i]=sc.nextInt();
        }
        int even=0;
        int odd=0;
        for (int i = 0; i <arr.length; i++) {
            if(arr[i]%2==0){
                even++;
                
            }
            else{
                odd++;
                
            }
            
        }
        System.out.print("even:"+even);
         System.out.print("odd:"+odd);
    }
}
